internal/engine/upper.go

```diff
@@ -218,29 +218,30 @@ func (u Upper) maybeStartBuild(ctx context.Context, st *store.Store) {
 		}
 
 		if newManifest.Equal(ms.Manifest) {
-			logger.Get(ctx).Debugf("Manifest %s hasn't changed, not rebuilding", ms.Manifest.Name)
-			matcher, err := ms.Manifest.ConfigMatcher()
+			logger.Get(ctx).Debugf("Detected config change, but manifest %s hasn't changed",
+				ms.Manifest.Name)
+
+			// TODO(maia): if ms.LastError is a bad-config error, clear it [ch603]
+
+			changedFilesWithoutConfigFiles, err := ms.PendingFileChangesWithoutConfigFiles(ctx)
```

Comment:
how do you know the config files aren't needed in the container?

Reply:
Legit. I'll make a ticket for fixing this down the line.

Reply:
[Tracked here](https://app.clubhouse.io/windmill/story/621/configfiles-not-copied-to-container)